bound/kt/src/test/kotlin/tbdex/sdk/httpclient/E2ePfiExemplarIntegrationTest.kt

```diff
@@ -1,15 +1,13 @@
 package tbdex.sdk.httpclient
 
-import com.fasterxml.jackson.databind.ObjectMapper
 import org.junit.jupiter.api.Assertions.assertNotEquals
-import org.junit.jupiter.api.Assertions.fail
+import org.junit.jupiter.api.Disabled
 import org.junit.jupiter.api.Test
 import tbdex.sdk.messages.*
-import tbdex.sdk.rust.RustCoreException
-import tbdex.sdk.rust.Web5RustCoreException
 import tbdex.sdk.web5.*
 
 class E2ePfiExemplarIntegrationTest {
+    @Disabled
```

Comment:
could add a line of documentation explaining that you have to spin up a tbdex-exemplar and then run this test